tests/test_ships.py

```diff
@@ -2,22 +2,21 @@
 from unittest.mock import patch, MagicMock
 from core.board import Board
 from core.ships import Ships
-from core.placement import Place
 
 class TestShips(TestCase):
-    def test_ships_is_initialized_with_a_five_unique_ships(self):
+    def test_all_ships_contains_all_5_ships(self):
         ships = Ships()
-        aircraft_carrier_symbol = 'AC'
-        battleship_symbol = 'B'
-        cruiser_symbol = 'C'
-        submarine_symbol = 'S'
-        destroyer_symbol = 'D'
-
-        self.assertEqual(ships.aircraft_carrier['symbol'], aircraft_carrier_symbol)
-        self.assertEqual(ships.destroyer['symbol'], destroyer_symbol)
-        self.assertEqual(ships.cruiser['symbol'], cruiser_symbol)
-        self.assertEqual(ships.submarine['symbol'], submarine_symbol)
-        self.assertEqual(ships.battleship['symbol'], battleship_symbol)
+        self.assertEqual(len(ships.all_ships), 5)
+
+    def test_each_ship_has_a_hit_location_array(self):
```

Comment:
maybe a better name for this test is `test_each_ship's_hit_location_array_is_empty_by_default`